### DISCUSSION TITLE
Jetson Nano Support

### DISCUSSION THREAD

**Discussion by SeeRich** [pythops/jetson-image#141] [General]
Can the jetson nano really use Ubuntu 20.04 and 22.04 as a base OS image? I was under the impression that it was only Ubuntu 18.04 because that's what comes with the Nvidia Installer SDK?

**Comment by pythops** [comment]
Yes it can.  You can try it yourself by building the image :)

**Reply by SeeRich** [reply]
This is great news! Fighting with 18.04 is not that fun. Just to make sure, even with 20.04, you can still make use of all the hardware accelerators (GPU, NVENC, NVDEC) and jetson multimedia libraries?

I will give this a try but there is quite a bit of setup involved to get my applications compiling and running with a 20.04 based image so I just wanted to see if you have already tested this.

**Reply by pythops** [reply]
Yes I have tested it and it works perfectly. Feel free to open issues/discussions if you encounter any problem or if anything is unclear

**Reply by nokados** [reply]
I use it with 22.04. Wi-Fi does not work

**Reply by pythops** [reply]
What do you mean by wifi does not work ?

**Reply by nokados** [reply]
The device is shown, but if I try to up it with:
```
ip link set wlan0 up
```
I see this in the journal:
```
iwlwifi 0000:01:00.0: L1 Enabled - LTR Enabled
iwlwifi 0000:01:00.0: L1 Enabled - LTR Enabled
iwlwifi 0000:01:00.0: L1 Enabled - LTR Enabled
iwlwifi 0000:01:00.0: L1 Enabled - LTR Enabled
IPv6: ADDRCONF(NETDEV_UP): wlan0: link is not ready
systemd-networkd: wlan0: Link UP
```
and `ip a` says it is down
